code/datums/elements/falling_hazard.dm

```diff
@@ -0,0 +1,70 @@
+/datum/element/falling_hazard
+	element_flags = ELEMENT_BESPOKE | ELEMENT_DETACH
```

Comment:
You don't need ELEMENT_DETACH unless you are cleaning up stuff that isn't automatically cleaned up when the host is destroyed.

https://github.com/tgstation/tgstation/pull/70972